src/andresmoschini/PrimeFactors.test.ts

```diff
@@ -0,0 +1,105 @@
+import { generate } from "./PrimeFactors";
+
+describe(generate.name, () => {
+  it.each([
+    {
+      scenario: "supported value",
+      n: 1,
+      expectedResult: [],
+    },
+    {
+      scenario: "supported value",
+      n: 2,
+      expectedResult: [2],
+    },
+    {
+      scenario: "supported value",
+      n: 3,
+      expectedResult: [3],
+    },
+    {
+      scenario: "supported value",
+      n: 4,
+      expectedResult: [2, 2],
+    },
+    {
+      scenario: "supported value",
+      n: 5,
+      expectedResult: [5],
+    },
+    {
+      scenario: "supported value",
+      n: 6,
+      expectedResult: [2, 3],
+    },
+    {
+      scenario: "supported value",
+      n: 7,
+      expectedResult: [7],
+    },
+    {
+      scenario: "supported value",
+      n: 8,
+      expectedResult: [2, 2, 2],
+    },
+    {
+      scenario: "supported value",
+      n: 9,
+      expectedResult: [3, 3],
+    },
+    {
+      scenario: "supported value",
+      n: 4620,
+      expectedResult: [2, 2, 3, 5, 7, 11],
+    },
+    {
+      scenario: "supported value (it is a prime number 🤦‍♂️)",
+      n: 1234568789,
+      expectedResult: [1234568789],
```

Comment:
I added this scenario from the beginning to make it explicit that I was not covering big numbers. But casually this number is prime, so it was always working fine 🤦‍♂️